### PR TITLE
Create G-factor speed.md

### PR DESCRIPTION
**Pull request by thehanggit**
complete the speed calculation documentation

### CODE REVIEW

File: docs/data/G-factor speed.md [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+
+| Variation | Baseline | Spatial | Temporal |
+| --------- | -------- | ------- | -------- |
+| 1         |    ✔️    |         |          |
+| 2         |          | detector|         |
+| 3         |          | station |         |
+| 4         |          | corridor |         |
+| 5         |          | detector | 4       |
+| 6         |          | station  | 4       |
+| 7         |          | corridor  | 2       |
```
Comment (mmmiah) [inline docs/data/G-factor speed.md:80]: line 80 last column should be 4 or 12?

File: docs/data/G-factor speed.md
Comment (mmmiah) [inline docs/data/G-factor speed.md:1]: line 160, #### MSE, MAE, and RMSE

can be revise as MAE, RMSE and MAPE

Comment (mmmiah) [inline docs/data/G-factor speed.md:1]: This MAE, RMSE and MAPE metric provide clear picture about the model performance. It seems that model is working pretty good in terms of prediction accuracy.

General comment:
Is the model developed and tested only for SR 91? If you are planning to test 2nd route, I would consider the longest route like 101 that pass through several districts and counties and should have much data variation to see how does the same model 8 works on it.

**Review by mmmiah — COMMENTED** [review]
Overall Looks good! Just address some minor issues that I pointed! Thanks!

File: docs/data/G-factor speed.md
Comment (mmmiah) [reply docs/data/G-factor speed.md:1]: @thehanggit , please response each of the above comments to describe how did you address the issues!

**Review by mmmiah — APPROVED** [review]

**Comment by thehanggit** [comment]
Hi @JamesSLogan, can you help me resolve this failure? Thanks!

**Comment by JamesSLogan** [comment]
As discussed in our session, feel free to merge even with the dbt cloud failure, since it isn't relevant to our caused by this pr 👍

**Comment by thehanggit** [comment]
> As discussed in our session, feel free to merge even with the dbt cloud failure, since it isn't relevant to our caused by this pr 👍

Thank you, James! Merged already

**Pull request merged**